### PR TITLE
Added Komoot presence

### PR DESCRIPTION
**Pull request by 7loud**
Proof:
Screenshot_429

Private tour:
Screenshot_430

Public tour:
image

Screenshot_431
Screenshot_432
...

### CODE REVIEW

**Comment by N0chteil** [comment]
> ## el mundo";
> https://github.com/PreMiD/Presences/blob/9a974212d58ff1b6a0c883e90a9e502f795f6c58/websites/K/Komoot/presence.ts#L63-L68
> 
> ###### This comment was generated by [todo](https://todo.jasonet.co) based on a `todo` comment in [9a97421](https://github.com/PreMiD/Presences/commit/9a974212d58ff1b6a0c883e90a9e502f795f6c58) in #4036. cc @N0chteil.

😂

File: websites/K/Komoot/presence.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        else if (url === "www.komoot.fr") return "visible par : tout le monde";
+        else if (url === "www.komoot.it") return "visibile a: tutti";
+        else if (url === "www.komoot.es") return "visible para todo el mundo";
+        else if (url === "www.komoot.nl") return "zichtbaar voor: iedereen";
+      })();
+
+    return document.querySelector(q).getAttribute(a).toLowerCase() === wl;
+  }
+
+  if (presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/K/Komoot/presence.ts:70]:
```suggestion
  if (!presenceData.details) {
```

**Review by Slowlife01 — COMMENTED** [review]
And would be good if you add support for `topics`

**Comment by N0chteil** [comment]
image

**Review by Slowlife01 — APPROVED** [review]

**Review by rxri — APPROVED** [review]

**Pull request merged**